CvPoint pt1
CvPoint pt2
CvColor color
int32 thickness
int32 lineType
int32 shift